### PR TITLE
styling for Initiate connection request form

### PR DESCRIPTION
**Pull request by Ellenn-A**
# Pull Request

## Checklist
- [x] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## PR Type

Please delete options that are irrelevant.

- [x] Bug Fix
- [x] Documentation Update
- [x] Code style update (formatting, local variables)

## Linked tickets

- https://digicatapult.atlassian.net/browse/VR-118
- https://digicatapult.atlassian.net/browse/VR-55

## High level description

Added styling and minor code changes, conserving functionality. 

## Detailed description
Added styling according to designs. 
Screenshot 2024-06-03 at 15 56 35
Screenshot 2024-06-03 at 15 56 55
Screenshot 2024-06-03 at 15 57 10

### CODE REVIEW

File: public/styles/main.css [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
   /* color pallete */
   --text-color: #45494c;
-  --text-color-secondary: #6E7079;
+  --text-color-secondary: #6e7079;
   --text-color-sub: #c9cace;
   --bg-color: #f4f5fb;
-  --secondary-color: #FFCC91;
+  --secondary-color: #ffcc91;
   --accent-color: #5670f1;
+  --light-green: #32936f;
```
Comment (mattdean-digicatapult) [inline public/styles/main.css:19]: is this used?

File: public/styles/main.css [hunk 2]
```diff
@@ -146,12 +158,12 @@ a.connections-table.icon {
   height: 14px;
   background-repeat: no-repeat;
   background-position: center;
-  background-color: transparent; 
-  background-image: url("/public/images/plus.svg");  
+  background-color: transparent;
+  background-image: url('/public/images/plus.svg');
 }
-
+/* unused??? */
```
Comment (mattdean-digicatapult) [inline public/styles/main.css:164]: if it's unused remove

File: src/views/newConnection.tsx [hunk 3]
```diff
@@ -26,7 +26,9 @@ export default class newConnectionTemplates {
         <div class="connections header">
           <span>Invite New Connection</span>
         </div>
-        <this.companyFormInput targetBox={targetBox} formStage={formStage} />
+        <div class="connections list ">
```
Comment (mattdean-digicatapult) [inline src/views/newConnection.tsx:29]: this looks a bit silly. I assume this wasn't mean to be two classes but instead one called `connection-list`?

File: src/views/newConnection.tsx [hunk 4]
```diff
@@ -43,64 +45,92 @@ export default class newConnectionTemplates {
...
-              <p>
-                {Html.escapeHtml(
-                  `Company House Number: ${params.targetBox.status === 'success' && params.targetBox.company.company_number}`
-                )}
-              </p>
-              <p>{Html.escapeHtml(`Email Address: ${params.targetBox.status === 'success' && params.email}`)}</p>
+          <this.stepper formStage={params.formStage} />
+          <div class="align-horizontally">
+            <div>
+              <div class="align-vertically ">
```
Comment (mattdean-digicatapult) [inline src/views/newConnection.tsx:51]: This is quite the div soup. I migth need you to walk me through why we need this.

File: public/styles/main.css [hunk 5]
```diff
@@ -91,6 +95,11 @@ a.connections-table.icon {
   font-size: 2rem;
   padding: 0.5rem;
 }
+.header.small {
```
Comment (mattdean-digicatapult) [inline public/styles/main.css:98]: I'm not sure I like how we're re-suing `small` here for heading and test. If there's some shared meaning for small (say `font-size: 0.8em`) then it should be defined in it's own small class. If small means something completely different in the two cases we should probably just have `heading-small` as a single class. I get you're just following patterns elsewhere, but this is the time these patterns become fixed so I'm being a bit strict

File: src/views/newConnection.tsx [hunk 6]
```diff
@@ -150,11 +185,32 @@ export default class newConnectionTemplates {
 
   public stepper = (params: { formStage: FormStage }): JSX.Element => {
     if (params.formStage === 'form') {
-      return <div>-----Step 1</div>
+      return (
+        <>
```
Comment (mattdean-digicatapult) [inline src/views/newConnection.tsx:189]: you don't need the fragments here as there is only one child (same with the others in this function), remove

File: seeds/connections-seed.cjs
Comment (mattdean-digicatapult) [inline seeds/connections-seed.cjs:1]: you can revert this change I fixed it in my PR #38

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

File: src/views/newConnection.tsx (hunk 3 above)
Comment (Ellenn-A) [reply src/views/newConnection.tsx:29]: this was sth Paul was using I left it be and created a new class `card-body` which I'll be using

File: public/styles/main.css [hunk 7]
```diff
@@ -251,3 +281,92 @@ a.connections-table.icon {
...
+}
+.bar-1-3::before {
+  width: 33.33%; /* 1/3 of the bar */
+}
+.bar-2-3::before {
+  width: 66.66%; /* 2/3 of the bar */
+}
+.bar-3-3::before {
+  width: 100%; /* 3/3 of the bar */
+}
```
Comment (mattdean-digicatapult) [inline public/styles/main.css:367]: one really small thing, it's up to you if you change. Can you change these to instead of changing width making them change the `transform` property with https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scaleX? This will make it smoother when we animate this (which I can definitely see us doing). See also https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin

**Review by mattdean-digicatapult — APPROVED** [review]
I think there's some more work to be done on styling but this is a massive improvement. Approving so we can keep moving
